onnxruntime/contrib_ops/cuda/bert/cutlass_fmha/fmha_launch_template.h

```diff
@@ -75,12 +75,8 @@ struct RightPaddingBatchHook {
           batch_id * lse_dim * p.num_heads + head_id * lse_dim + query_start;
     }
 
-    // Custom masking
-    if (p.causal_diagonal_ptr) {
```

Comment:
custom masking been removed?

Reply:
Right. It was removed in cutlass, and it is not used in our code base so I think it is fine.